### PR TITLE
Active Page: Collapse/Expand all buttons

### PR DESCRIPTION
**Pull request by cannarelladev**
# Description

Added user setting memory Active Page and Collapse/Expand all buttons

### CODE REVIEW

File: frontend/src/components/activePage/TableTemplate/TableTemplate.tsx [hunk 1]
```diff
@@ -65,6 +90,25 @@ const TableTemplate: FC<ITableTemplateProps> = ({ ...props }) => {
...
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [expandedId]);
+
+  useEffect(() => {
+    if (collapseAll) {
+      collapseTemplate();
+    }
+    if (expandAll) {
+      expandTemplate();
+    }
```
Comment (QcFe) [inline frontend/src/components/activePage/TableTemplate/TableTemplate.tsx:108]:
```suggestion
    if (collapseAll) collapseTemplate();
    if (expandAll) expandTemplate();
```
stay compact, check other occurrences

File: frontend/src/components/activePage/TableWorkspace/TableWorkspace.tsx [hunk 2]
```diff
@@ -8,11 +8,34 @@ import TableWorkspaceRow from './TableWorkspaceRow';
...
+      .getItem('prevExpandedIdActivePageWorkspace')
+      ?.split(',') ?? ['']
+  );
+
+  const expandWorkspace = () => {
+    setExpandedId(workspaces.map(ws => ws.id));
+  };
+
+  const collapseWorkspace = () => {
+    setExpandedId(['']);
```
Comment (QcFe) [inline frontend/src/components/activePage/TableWorkspace/TableWorkspace.tsx:37]: I'm probably missing something. Why not this?
```suggestion
    setExpandedId([]);
```

File: frontend/src/components/activePage/ActiveView/ActiveView.tsx [hunk 3]
```diff
@@ -60,6 +69,54 @@ const ActiveView: FC<IActiveViewProps> = ({ ...props }) => {
...
               />
+              <Button
+                className="hidden xl:block"
+                type="primary"
+                shape="round"
+                size="middle"
+                icon={<FullscreenOutlined />}
+                onClick={() => {
+                  setExpandAll(true);
+                }}
```
Comment (QcFe) [inline frontend/src/components/activePage/ActiveView/ActiveView.tsx:80]:
```suggestion
                onClick={() => setExpandAll(true)}
```
Also on the other cases below

**Review by QcFe — APPROVED** [review]
Minor style comments and a question.

File: frontend/src/components/activePage/TableWorkspace/TableWorkspace.tsx (hunk 2 above)
Comment (cannarelladev) [reply frontend/src/components/activePage/TableWorkspace/TableWorkspace.tsx:37]: i don't know why until now i had to use that notation. but now i fix it and remove ''

**Pull request merged**